src/classes/frModel.cls

```diff
@@ -214,10 +217,7 @@ public with sharing abstract class frModel {
             }
             record.put(field, value);
         } catch (Exception ex) {
-            insert new Error__c(Error__c = 'Field mapping exception. Object type: '+ record.getSObjectType().getDescribe().getName()
-                                +' Record Id: '+record.Id+' - Funraise Id: '+ funraiseId + ' - Field: '+field.getDescribe().getName()+' - Value: '+value
-                                +' Exception: '+ex
-                               );
+            System.debug(ex);
```

Comment:
This change should be reversed, the error should be logged in the custom object and not debugged

Reply:
Added back this with an exception to just debug it as i was running into some null pointer errors in some tests files.